### ISSUE TITLE
Keyboard Navigation (Accessibility Issue) Not working.

### ISSUE THREAD

**Issue by xgpt** [laurent22/joplin#12234]
### Operating system

Linux

### Joplin version

Joplin 3.3.12 (prod, linux)

### Desktop version info

Joplin 3.3.12 (prod, linux)

Device: linux, AMD Ryzen 5 7640U w/ Radeon 760M Graphics
Client ID: 4071998e7c1146e6bedca82f843c1cc8
Sync Version: 3
Profile Version: 47
Keychain Supported: No
Alternative instance ID: -

Revision: 4d790b6ff

Backup: 1.4.3
Freehand Drawing: 3.0.1

### Current behaviour

1. alt-F (open file menu) WORKS!
2. press arrow keys Up/Down (Navigates up and down through file menu, WORKS!)
3. Press arrow keys left/right (Does not move between menus as expected, stuck in "File" menu, does not move to Edit/View/Go/Notebook/Note/Tools/Help, BROKEN!)

### Expected behaviour

Expected behavior is to be able to navigate through all menu options with just arrow keys. This is the norm that I'm used to and is very useful when using a keyboard centric navigation or keyboard centric accessibility scheme.

**Comment by tomasz1986** [comment]
I can confirm the problem of being unable to move between different menus with arrow keys. Just as a workaround, you can still access them, e.g. by pressing `Alt+E` to go to "Edit", `Alt+V` to go to "View", and so on.

**Comment by personalizedrefrigerator** [comment]
I'm linking to an upstream Electron issue that discusses a similar issue — https://github.com/electron/electron/issues/2504 (see [comment](https://github.com/electron/electron/issues/2504#issuecomment-1546281128)).

> Just as a workaround, you can still access them, e.g. by pressing Alt+E to go to "Edit", Alt+V to go to "View", and so on.

As another workaround, pressing <kbd>alt</kbd> once moves focus to the toplevel menubar (testing on Fedora 42 Linux). After doing this, pressing <kbd>left</kbd> and <kbd>right</kbd> move focus between "File", "Edit", "View", etc.

**Comment by personalizedrefrigerator** [comment]
Expected behavior is [documented in the ARIA Authoring Practices Guide](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/#keyboardinteraction).

**Edit**: For reference, I'm linking to [an upstream pull request](https://github.com/electron/electron/pull/15302) that previously improved the accessibility of the Electron menubar. Fixing this issue may involve creating an upstream pull request that modifies similar files.
**Edit 2**: The upstream keyboard management code was [refactored here](https://github.com/electron/electron/pull/29262) to use Chromium's [AccessiblePaneView](https://github.com/chromium/chromium/blob/e07cf5c8c45818f689c51bceb651b6cdec46aa92/ui/views/accessible_pane_view.cc#L180) for focus handling.
